extra-security/clipass/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=clipass
+PKGSEC=security
+PKGDEP="nodejs"
+BUILDDEP="nodejs"
```

Comment:
If `nodejs` has been specified as `$PKGDEP`, you won't need to specify it again in `$BUILDDEP`.

Reply:
Does it mean that all PKGDEP will be automatically installed when building? I interpreted it as a runtime dependency declaration, while it is both runtime and buildtime? This is big news to me.

Reply:
> Does it mean that all PKGDEP will be automatically installed when building? I interpreted it as a runtime dependency declaration, while it is both runtime and buildtime? This is big news to me.

Yep, both `$PKGDEP` and `$BUILDDEP` will be installed on build-time.

Reply:
Ack.